src/renderer/component/walletSendTip/view.jsx

```diff
@@ -1,66 +1,92 @@
-// I'll come back to this
-/* eslint-disable */
+// @flow
 import React from 'react';
-import Link from 'component/link';
-import { FormRow } from 'component/common/form';
+import Button from 'component/link';
+import { FormField } from 'component/common/form';
 import UriIndicator from 'component/uriIndicator';
 
-class WalletSendTip extends React.PureComponent {
-  constructor(props) {
+type Props = {
+  claim_id: string,
+  uri: string,
+  title: string,
+  errorMessage: string,
+  isPending: boolean,
+  sendSupport: (number, string, string) => void,
+  onCancel: () => void,
+  sendTipCallback?: () => void,
+};
+
+type State = {
+  amount: number,
+};
+
+class WalletSendTip extends React.PureComponent<Props, State> {
+  constructor(props: Props) {
     super(props);
 
     this.state = {
-      amount: 0.0,
+      amount: 0,
     };
+
+    (this: any).handleSendButtonClicked = this.handleSendButtonClicked.bind(this);
```

Comment:
I dont think it really matters, but isn't the better way to do this to make a type for WalletSendTip?

Reply:
Not sure I follow